src/isIsogram.test.js

```diff
@@ -6,4 +6,28 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
```

Comment:
Consider adding a test case for an empty string, as the task description specifies that an empty string should be considered an isogram.